theme/theme.ts

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from "react-native";
```

Comment:
there's no need for this right now. I'll move styles to a common theme file when they are actually used at several places.